### DISCUSSION TITLE
Available on Maven Central

### DISCUSSION THREAD

**Discussion by madsop-nav** [openhtmltopdf/openhtmltopdf#36] [Announcements]
I'm happy to announce that we have now released this community maintained version of Open HTML to PDF to Maven Central.

Every push to the main branch will now automatically trigger a new release to Maven Central, as well as a GitHub release and also a deploy to the GitHub Package Registry.

The version number for the new artifacts is determined in the build process and is fully automated.

The first versions are already released and available, under the groupid `io.github.openhtmltopdf`.

Changes in each upcoming release can be seen in GitHub Releases.

**Comment by burka** [comment]
Yeah :tada: that rocks! :rocket:
